plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -6,54 +6,60 @@ import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-export default function Breadcrumbs({ item, children, renderStates }) {
-  function getBreadcrumb(item, id = "", name = "") {
-    const isDetailPage = [...item.path, item.name].join(".") === id;
-    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
-
+export default function Breadcrumbs({ path, name, children, extra }) {
+  /**
+   * @param  {string} link
+   * @param  {string[]} pathname path including name (if given)
+   * @param  {JSX.Element} [extra=null] extra JSX Components to render
+   *
+   * @return {JSX.Element} Breadcrumb Part Component
+   */
+  function buildJSX(link, pathname, extra = null) {
     return (
-      <Breadcrumb key={id} title="Jobs">
+      <Breadcrumb key={pathname.join(".")} title="Jobs">
         <BreadcrumbTextContent>
-          <Link to={link}>{name === "" ? "Jobs" : name}</Link>
+          <Link to={link}>{pathname.slice(-1)}</Link>
         </BreadcrumbTextContent>
 
-        {isDetailPage ? renderStates(item) : null}
+        {extra}
       </Breadcrumb>
     );
   }
 
-  function getBreadcrumbList(item) {
-    if (item == null) {
-      return [];
-    }
+  // we're starting from the back, so first we have children
+  let breadcrumbs = React.Children.toArray(children);
```

Comment:
If you have t explain this, why don't we work the other way around?

Reply:
I dont _have to_ :trollface: - started from the end because otherwise I would have to declare an empty array manually only to concat it with another empty array later…